include/onnxruntime/core/graph/graph.h

```diff
@@ -80,6 +80,19 @@ class Node {
   };
 
   explicit Node() = default;
+
+#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)
+  Node(const std::string& name,
+       const std::string& op_type,
+       const std::string& description,
+       const std::vector<NodeArg*>& input_args,
```

Comment:
Same as above - the constructor is a just thin bridge.

Reply:
Switched to gsl::span, thanks!

Reply:
The comment and the standard says to pass gsl::span by value.

Reply:
Done.

Reply:
This would only work for non-const containers, but the original idea was to pass any container const and non-const. So you want those to be `gsl::span<NodeArg* const>`

Reply:
Yes, we should do that.
Switch to NodeArg* const instead.